src/state/ducks/todos/types/api.ts

```diff
@@ -0,0 +1,4 @@
+export interface ApiRequestPayload {
+  title: string;
+  completed: boolean;
+}
```

Comment:
Made this separate because ApiRequestPayload can be changed it can take other things as well that are different from ITodo .So making it dependent on ITodo may be a problem.